x-pack/metricbeat/modules.d/enterprisesearch-xpack.yml.disabled

```diff
@@ -0,0 +1,11 @@
+# Module: enterprisesearch
+# Docs: https://www.elastic.co/guide/en/beats/metricbeat/master/metricbeat-module-enterprisesearch.html
+
+- module: enterprisesearch
+  xpack.enabled: true
+  metricsets: ["health", "stats"]
```

Comment:
Do we need the metricsets? The elasticsearch module specifies a locked metricset when running in xpack mode https://github.com/elastic/beats/blob/a78436b3b986da8957451ce767721822cf4129c7/metricbeat/module/elasticsearch/elasticsearch.go#L47

Seems to work find w/o the list in my testing. Maybe because it just enables all of them.